tests/integration.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
+mod tests {
+    use anyhow::Result;
+    use image_rs::image::ImageClient;
+    use std::path::Path;
+    use std::process::{Child, Command};
+
+    fn start_attestation_agent() -> Result<Child> {
+        let script_dir = format!("{}/{}", std::env!("CARGO_MANIFEST_DIR"), "scripts");
+        let aa_path = format!("{}/{}", script_dir, "attestation-agent");
+
+        if !Path::new(&aa_path).exists() {
+            let script_path = format!("{}/{}", script_dir, "build_attestation_agent.sh");
+            Command::new(script_path)
+                .output()
+                .expect("Failed to build attestation-agent");
+        }
+
+        Ok(Command::new(aa_path)
+            .args(&["--keyprovider_sock"])
+            .args(&["127.0.0.1:48888"])
+            .spawn()?)
+    }
+
+    #[tokio::test]
+    async fn test_image_rs() {
+        let mut aa = start_attestation_agent().expect("Failed to start attestation agent!");
+
+        let manifest_dir = std::env!("CARGO_MANIFEST_DIR");
```

Comment:
nit: you could have a `fn manifest_dir()` at test module level so that the function could be called here and in `start_attestation_agent()`